script.js

```diff
@@ -1,5 +1,37 @@
 /* eslint-disable max-classes-per-file */
```

Comment:
- [ ] Let's try to not disable linters as they help us write better and cleaner code